src/components/Chat/ChatItem.tsx

```diff
@@ -63,12 +63,13 @@ class ChatItem extends UIComponent<ReactProps<ChatItemProps>, any> {
 
   private renderChatItem(styles: ComponentSlotStylesPrepared) {
     const { message, gutter, gutterPosition } = this.props
-    const gutterElement = gutter && Slot.create(gutter, { defaultProps: { styles: styles.gutter } })
+    const gutterElement =
```

Comment:
Although this produces an element, we wanted to avoid `*.createElement()` since it is quite different from the `React.createElement()`.  For all other components, I think the reading makes more sense:

```
Button.create()
Image.create()
...
```

It seems the `Component.create()` is a little more ambiguous since `Component` is so abstract.  It sort of reads like `createComponent()`.

Let's leave it as is for now and see if we get more feedback.

Reply:
what about using `Box` as suggested previously by @mnajdova and @layershifter 
As I remember, @layershifter mentioned `Box` is used successfully in other UI libraries as well

I think this would remove confusion. As we remember, the semantical problem `Slot` has is that it's describing an entity that is part of something, but we're using `Slot` for wrapping components as well (opposite meaning almost).

What do you think @kuzhelov ?

Reply:
@Bugaa92, `Box` is a much better alternative, would totally support this move

Reply:
renamed to `Box`